content/docs/handling-events.md

```diff
@@ -91,13 +97,17 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](http://codepen.io/gaearon/pen/xEmzGg?editors=0010)
+[**Probeer het op CodePen**](http://codepen.io/gaearon/pen/xEmzGg?editors=0010)
 
-You have to be careful about the meaning of `this` in JSX callbacks. In JavaScript, class methods are not [bound](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_objects/Function/bind) by default. If you forget to bind `this.handleClick` and pass it to `onClick`, `this` will be `undefined` when the function is actually called.
+Je bent best voorzichtig met de betekenis van `this` in JSX callbacks.
```

Comment:
❗️ 
```suggestion
Je moet voorzichtig zijn met de betekenis van `this` in JSX callbacks.
```

Reply:
Well, I thought I'd be gentle 🤷‍♂️

Reply:
It's not necessarily about being harsh or gentle. The original sentence is a warning to the user regarding the meaning of `this` in callbacks. That alerting meaning is lost in your translation.